changelogs/unreleased/fiber-basic-api.md

```diff
@@ -0,0 +1,4 @@
+## feature/fiber
+
+* Exported `fiber_set_name_n`, `fiber_get_name`, `fiber_get_id`,
+  `fiber_get_csw`, and `fiber_find` FFI API functions.
```

Comment:
I would say 'exported into the public C API and usable with LiaJIT FFI as well'.

Reply:
done